fabtests/pytest/efa/test_rdm.py

```diff
@@ -7,9 +7,9 @@
 @pytest.mark.parametrize("iteration_type",
                          [pytest.param("short", marks=pytest.mark.short),
                           pytest.param("standard", marks=pytest.mark.standard)])
-def test_rdm_pingpong(cmdline_args, iteration_type, completion_semantic, memory_type):
+def test_rdm_pingpong(cmdline_args, iteration_type, completion_semantic, memory_type, completion_type):
     efa_run_client_server_test(cmdline_args, "fi_rdm_pingpong", iteration_type,
-                               completion_semantic, memory_type, "all")
+                               completion_semantic, memory_type, "all", completion_type=completion_type)
```

Comment:
Rebase?

Reply:
not sure I follow, why rebase?

Reply:
nvm I thought this was from a previous commit.